packages/react/src/types.ts

```diff
@@ -31,5 +31,65 @@ export type CssObject<TProps> = Readonly<{
 export type CssFunction<TProps = unknown> =
   | CssType<TProps>
   | BasicTemplateInterpolations // CSS values in tagged template expression
+  | null
   | boolean // Something like `false && styles`
   | undefined; // Something like `undefined && styles`
+
+// List of pseudo-classes and pseudo-elements are from csstype
+// but with & added in the front, so that we target the current element
+// (instead of a child element)
+
+// We also exclude anything that requires providing an argument
+// (e.g. &:not(...) ), and anything that uses information from elements
+// outside of the current element (e.g. &:first-of-type)
+export type CSSPseudos =
```

Comment:
Moved from `pseudos.ts` to here

Reply:
/nit could use css type as this:

```js
import type { SimplePsuedos } from 'csstype';

export type CSSPseudos = `&${Exclude<
    SimplePseudos,
    ':first' | ':first-child' | ':first-of-type' | ':last' | ':last-child' | ':last-of-type' | ':only-child' | ':only-of-type'
>}`;
```

Reply:
We definitely can — instead we're opting to explicitly define the psuedos so we can handle them as appropriate. E.g. there might be new pseudos we don't want to support, or something we want to delay.